### ISSUE TITLE
🐞 beta.97 | breaking AOT Compiler on SvelteKit

### ISSUE THREAD

**Issue by oskar-gmerek** [master-co/css#112]
### What happened?

After the update to beta.97 SvelteKit doesn't start and breaks with the error `Error: Dynamic require of "path" is not supported`.

### Version

2.0.0-beta.97

### Relevant log output

```shell
failed to load config from /Users/x/Desktop/github/x/vite.config.js
error when starting dev server:
Error: Dynamic require of "path" is not supported
    at file:///Users/x/Desktop/github/x/node_modules/.pnpm/techor@1.0.5/node_modules/techor/dist/index.mjs:1:382
    at file:///Users/x/Desktop/github/x/node_modules/.pnpm/techor@1.0.5/node_modules/techor/dist/index.mjs:1:994
    at file:///Users/x/Desktop/github/x/node_modules/.pnpm/techor@1.0.5/node_modules/techor/dist/index.mjs:1:462
    at file:///Users/x/Desktop/github/x/node_modules/.pnpm/techor@1.0.5/node_modules/techor/dist/index.mjs:2:62587
    at ModuleJob.run (node:internal/modules/esm/module_job:193:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:530:24)
    at async loadConfigFromBundledFile (file:///Users/x/Desktop/github/x/node_modules/.pnpm/vite@4.0.3/node_modules/vite/dist/node/chunks/dep-0bae2027.js:62058:21)
    at async loadConfigFromFile (file:///Users/x/Desktop/github/x/node_modules/.pnpm/vite@4.0.3/node_modules/vite/dist/node/chunks/dep-0bae2027.js:61943:28)
    at async resolveConfig (file:///Users/x/Desktop/github/x/node_modules/.pnpm/vite@4.0.3/node_modules/vite/dist/node/chunks/dep-0bae2027.js:61564:28)
 ELIFECYCLE  Command failed with exit code 1.
```

**Comment by 1aron** [comment]
@oskar-gmerek We also found this problem and are urgently fixing it!

**Comment by 1aron** [comment]
The bug has been fixed in [v2.0.0-beta.98](https://github.com/master-co/css/releases/tag/v2.0.0-beta.98)

![image](https://user-images.githubusercontent.com/33840671/209446470-9e68003b-3696-4591-be89-f99c885616de.png)
